### PR TITLE
chore: Skip generated lint files (swagger and routes) 

### PR DESCRIPTION
**Pull request by rephus**
Closes: #5930

### Description:

https://eslint.org/docs/latest/use/configure/ignore

Before:

Screenshot from 2023-06-20 11-46-13

Now:

image

### CODE REVIEW

File: packages/backend/src/generated/routes.ts [hunk 1]
```diff
@@ -434,19 +434,13 @@ const models: TsoaRoute.Models = {
...
-    },
-    // WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
     OrganizationMemberProfileUpdate: {
         dataType: 'refAlias',
         type: {
-            ref: 'Partial_Pick_OrganizationMemberProfile.role__',
+            dataType: 'nestedObjectLiteral',
+            nestedProperties: {
+                role: { ref: 'OrganizationMemberRole', required: true },
+            },
```
Comment (joaoviana) [inline packages/backend/src/generated/routes.ts:443]: these changes are from this PR right? https://github.com/lightdash/lightdash/pull/5954

**Review by joaoviana — APPROVED** [review]

File: packages/backend/src/generated/routes.ts (hunk 1 above)
Comment (owlas) [reply packages/backend/src/generated/routes.ts:443]: yep make sure it's rebased on `main` ! Maybe something changed here

File: packages/backend/package.json [hunk 2]
```diff
@@ -105,7 +105,7 @@
         "scheduler": "node dist/scheduler.js",
         "scheduler-dev": "ts-node-dev --rs --respawn -- src/scheduler.ts",
         "test": "TZ=UTC jest",
-        "linter": "eslint -c .eslintrc.js --ignore-path ./../../.gitignore",
+        "linter": "eslint -c .eslintrc.js --ignore-path ./../../.gitignore  --ignore-pattern ./src/generated/*",
```
Comment (owlas) [inline packages/backend/package.json:108]: :chef-kiss:

File: packages/backend/src/generated/routes.ts (hunk 1 above)
Comment (rephus) [reply packages/backend/src/generated/routes.ts:443]: Yes, these are the files "unlited", I rebased from master just now

**Comment by rephus** [comment]
For some reason Render didn't trigger, I've created it manually

**Pull request merged**